src/data/constants.js

```diff
@@ -192,6 +192,8 @@ export const Routes = {
     ],
     PAGE: `${BASE_PATH}/:page`,
     PAGES: {
+      // The category page just uses the posts path since a category in itself doesn't have posts
+      category: `${BASE_PATH}/topics/:topicId/posts/:postId`,
```

Comment:
This URL will display the details of a post related to a category `show all` posts. It will change the breadcrumb value and redirect the user to that specific topic post. Which is not a good UX experience. It should display post details while displaying the show all posts related to the category.
https://www.loom.com/share/b00e360a12934469b482880326ac80d3

Reply:
I will look into creating a separate path for this then. Last time I tried I had some issues, will see if I can get to it now.

Reply:
@xitij2000 You can complete this issue on this PR because the discussions MEF pipeline is broken right now and DevOps is working on it. 
I was testing on your latest routing fix. I'm facing really annoying behavior. Category `show all` filter is working fine but  Category  `other topics filter` is not working fine and displaying all posts for a specific topic.
https://www.loom.com/share/24f1dd6d04e0448f8ba316b4fcac4e6a

Reply:
Unfortunately I have been unable to reproduce this issue so far.

Reply:
@xitij2000 I tested the changes after your last force push. It is working fine now. Thanks for the update.